relevanceai/workflows/cluster_ops/ops.py

```diff
@@ -1525,3 +1525,80 @@ def update_documents_within_clusters(
         return results
+
+    def show(
+        self,
+        dataset_id: str,
+        field: str,
+        vector_field: str,
+        alias: str,
+        preview_num: int = 10,
+    ):
+        """
+        Shows the values of a clustering.
+
+        dataset_id: str
+            The dataset ID of with the clustering of interest.
+
+        field: str
+            The field whose values are of interest.
+
+        vector_field: str
+            The vector field that was used for the clustering.
+
+        alias: str
+            The alias of the clustering.
+
+        preview_num: int
+            The maximum number of values to preview. If a cluster has fewer
+            values than preview_num, the cluster list will be extended by
+            the appropriate number of np.nans to make up the difference.
+        """
+        if type(preview_num) is not int and preview_num <= 0:
+            raise TypeError(
+                f"Please provide a valid non-zero integer for {preview_num}."
+            )
+
+        if not vector_field.startswith(field):
+            raise ValueError(f"{vector_field} must be a vector of {field}.")
+
+        # Since there is no gaurantee self.dataset_id will be not None, it is
+        # safer to force the user to specify the dataset_id
+        ds = Dataset(
+            self.project, self.api_key, dataset_id, self.firebase_uid, fields=[]
+        )
+        schema = ds.schema
+
+        if field not in schema:
+            raise ValueError(f"{field} does not exist")
+
+        cluster_field = ".".join(["_cluster_", vector_field])
+        if cluster_field not in schema:
+            raise ValueError(f"{vector_field} has not been clustered yet.")
+
+        alias_field = ".".join([cluster_field, alias])
+        if alias_field not in schema:
+            raise ValueError(f"A clustering with alias {alias} has not been made yet.")
+
+        cluster_values = ds.to_pandas_dataframe(
+            select_fields=[alias_field, field], show_progress_bar=False
+        )
+        # converts {"vector_field": {"alias": "cluster-k"}} to "cluster-k"
+        cluster_values["_cluster_"] = cluster_values["_cluster_"].apply(
+            lambda cluster: cluster[vector_field][alias]
+        )
+
+        cluster_groups = dict(list(cluster_values.groupby("_cluster_")[field]))
+        clusters = {}
+        for cluster, values in cluster_groups.items():
+            length = min(len(values), preview_num)
+            # It is faster to index a list than a pandas Series
+            clusters[cluster] = values.tolist()[:length]
+
+            # If the number of values in the cluster is fewer than the desired
+            # preview number, insert np.nan to make up the difference. Note
+            # that the range of a negative number is not a range.
+            difference = preview_num - len(clusters[cluster])
+            clusters[cluster].extend([np.nan for _ in range(difference)])
+
+        return pd.DataFrame(clusters)
```

Comment:
This wouldn't work for images. You need to use `show_json`. 
Can we use the `operate` function here to operate across each cluster? Ideally we improve the `operate` backbone.